tests/fixtures/certificates/server.key

```diff
@@ -1,28 +1,27 @@
-boNAjY8/8nZHttt7KSbKa6uVOjzH9ZQjS2w5+M53LVflY0TOEAsoxm2NJSeVuONE
-3zKH75TMtH7JQmTEMt9iKPwP1uZ39dqJ//cwvNKALhrYRYB7cMmdGVx4L6Q6DT8W
-KYxzqwc4miRpQMQYQ0ChhsjH0dLLTiGwUdEiYwNdK/xJPmF6JBj966LbDZBeArGU
-X9ku1x3WmHF0Y7jPwalDao5jpB9vsgd3IQgovrZYgIcOMWFFjKHOQ/2Xzfg4C2EP
-345teIaXY9/6vGe8WsXNQcQbOYnkiAYpY64gizpSxQKBgQDyh1jCFuXztK1XoaFk
-io3IruxVLTKhyvc+ig/uXojxYUufatAPaNGeti9HQQK+/IExuPQN72NuyV9KSjH4
-tpg19+G0bXQe0osxw221VWlG5euQVBVTrX7+I1C6yxZShCk4GVba/OzntctOWDjT
-6lwFIHKVhbAGBpBDV/E6wv7XCwKBgQDoST9Er7/umP6oYKg3Bhscv17BugI5gIik
-pskFtstZEeI3KscnUbLZ3h/LPJYf2xh39GNaaFo9GLS9Pl+wQsbOOu1jRCiKV7ZC
-6zp1biaUXjo/j6izsaoh3QUYKCYqDHtxVn47uAhRjxT0q9EM8NmsLY2tv4G0AbQd
-uCEQR8CnfwKBgFAYuwt+nHscFJN4YeDekWzRuGN5BdvBOF6DIjfLKNbvFxaTJBuy
-tmgAKyURJ2MD6XcOaiuf/+q6vqm+Tzqgsyw/rV71NwBm8GmC7VyR1K8BiAamflSg
-W+x6pVLLd5fQ5fwNAk5YcbUkqPw6zqnOII3g5XbULAoRaNuU06pgc+F5AoGAM6CE
-BVoiiH9MqlPBT2l2AS+vedNDx1AADshbCYAVNhi5dJV3ag/Ido/d9SfkHfRuPcr9
-ggyLceTZ1u3UCrLsyM1kEZoGgKj6A2mQtqTiwjsnKxVc44NrUtdy2GmkAdiAXzJg
-IBnJD4eDmtXNQtkNKC1/vcxAoad3RtuzzZCFCzUCgYAv+2sRcpYtMWpdsViaSKzw
-YwIzWGNEJOOio59MHgOy0dvTxryEOLTk4sIZRyQmsFSQksm8PTCMEDoMH36t0F4o
-uEzXrjsxgxsZzkWeRJ+u2GifpWSBsw/cSxxscdpROn4d6f1/9DQf/TRhCDMz7Lir
-jKZQY0uSNHhWnzC/sF/pFQ==
------END PRIVATE KEY-----
+-----BEGIN RSA PRIVATE KEY-----
+MIIEowIBAAKCAQEA6Rlrujp4Za4xPTP+ipiqsC8S4wWhHA70zHD3PbYVs37QHi7u
+1koyWIi8FBda/mTQU5Q0glMYb1b8MeWO+O6TItzN/uqsJn10HLkD2k2WritavO8w
+ie/qIzgJkTYNkK/o4ZFqtDi7ylFR9oQ3dMlVIttqeZ8xcAk/lvakxqCaqkmdI0rl
+ilvQrq1GUOoE0zWYrLNORZBRpFIxSTZotRO1tVm6Hqe05fFy4H2dRnkw8VerWeAn
+gnYfhunHHIwI1lRDn8Kn2L7lw+8sv69rqB8qDN/EqxLUpKLM5KHZJkRdDK9nIR/M
+tbn94iSZ6abB/u6bIATeXVcSIHZpZSB6ASIeVwIDAQABAoIBAQCqpIJ5tZ44Cbpg
+LLUV70ddbUitS4yzBrvtAnHtCmR+n8JppkAQPgb3SzoslyB1BYaA+Qkrsvs7XvmG
+6Z+t2rF6Hxzqlg9qqgKbmFIauRwq6vAlShFFmY3VNIvEc0ag+jZJE3SC9Nl89eVD
+y2zxoQYjW29qE8FUXGDh+jOMeTxgldeDS6iyqG/BUILhlwh0skaUrGaLxsmTgMr5
+WZJEn13EZU6rO7GQl3+OxdHkCkorRYZ3md1k2viJTsjQfWT3MDA8PX6IcE1ueWrl
+Ght5VVPRPbTYcjhixxqNO32NayRxo4h2yadJtJL5xvY1Sg/Pb0yJ/LN5itg1oBF8
+y9Og53P5AoGBAPzuoeCSSRkKhhLKrYYjmzd4v1HKlf6af5sJvTetdCM/GEkmpw2K
+0Am4Tqm6VdRKKjYRDoGqMe4lUl2aoaMjwKBVM7YnQ56g9Fdg+ioWJKpkPgu46wDQ
+v19m9cSsGKc4LyEJziSXntOmKtRDY+ZmUY3L6D6OnVoCEGMzkg5MlTWdAoGBAOvt
+NNdS+t56JyWY3qRDQlzavg+B2ULlT/5oukPGEukflvbvHyeeW2ff+eNB9+JMTxox
+3hSadXUDG20/GTZAyF7FUpiuKdu0fCmt+QC7Qg1ZqcCjs541FDpdruzXvLl4Z1rJ
+jDkY31HAAH7QbeQm+tM8NglgdoJ5ew7DhJb5truDAoGAEL/s0GZfmNztjAkH/L4W
+5VGFufDkhfvNP4MhmLJP4pWnEhERNwBeToZ6Dxtb27cdnH78qvGOZOrrzW8UDS2s
+7/E0f3eSLssR40d0yZy1yOLiiPwx89ffFH5JAEE1UL2TwWhY0muIBRQX+f6MexGD
+jRRXyOzA1tkUJL2hinjf2LECgYBLCF7U2Gw202yd1XeBkG6Lr3ng3rAlSV7ej5ol
+sdoT0Fsn37bt17xEVgRYVYOrsg86Sn/R16nhdAKGBRpUmdff5Cu3qTRgzo7qhoz5
+wZRpQInvXWDjwQfU5DSc2KQJ7JQaWJ1DD3j+u+ysaEd61zbQdYbOBqLKLi6H7IC/
+r12BVwKBgF6FpUzb2QOwE9IvWKmrbFekmQjZGc3m3dPBxRVr45heBA5rpMMeWo9s
+pmtlnpepnDHvvdsNm0HTjZ+YpTnkt2kv7lHUzDB9lixnVb1vAmRi5sr9AjxVZGEx
+I5vTLKelvcJrx6ProvrXDdwzlgdn9yZen0zsNGtcjWtcbNVy/YV4
+-----END RSA PRIVATE KEY-----
```

Comment:
The files below were included in the repo but they really didn't need to be there as they are tests for a 3rd party we use.